### ISSUE TITLE
[Very Minor] Advanced solar panel orientation after spin thrusters used on space station

### ISSUE THREAD

**Issue by dj3520** [micdoodle8/Galacticraft#2118]
On space stations, if a spin thruster is used to change the orientation of the sunrize, advanced solar panels no longer point themselves at the sun.
![2016-02-27_15 12 17](https://cloud.githubusercontent.com/assets/8284194/13375101/e9258df8-dd64-11e5-9a3c-f5feacb2adf3.png)
`</nitpicking>`

**Comment by EzerArch** [comment]
Well, these solar panels would suit better on ground, not in a space station. We suggested that space stations should get [their own kind of solar panel](http://forum.micdoodle8.com/index.php?threads/solar-panel-arrays-for-space-stations.1977/).

**Comment by dj3520** [comment]
@EzerArch I've commented on that thread and a bit excited to see solar panels that look like those on satellites. The integration with Spin Thrusters would be something interesting as well since they can be redstone controlled. (And with a mod, computer controlled.)

Yet there's an environmental bonus on space stations that I've cranked up, meaning these are actually viable power sources in combination with either a large energy storage or a nighttime generator. Plus what's to say that a player that has had an advanced solar panel they don't use anymore? Might as well put it on a space station.
